.github/workflows/clang-tidy-nvidia.yaml

```diff
@@ -5,33 +5,36 @@ on: [push, pull_request]
 env:
   OPEN_UCX_LINK: https://github.com/openucx/ucx
   OPEN_UCX_BRANCH: master
-  HPCX_LINK: https://content.mellanox.com/hpc/hpc-x/v2.17.1rc2/hpcx-v2.17.1-gcc-mlnx_ofed-ubuntu20.04-cuda12-x86_64.tbz
-  CLANG_VER: 12
-  MLNX_OFED_VER: 5.9-0.5.6.0
-  CUDA_VER: 11-4
+  HPCX_LINK: https://content.mellanox.com/hpc/hpc-x/v2.17.1rc2/hpcx-v2.17.1-gcc-mlnx_ofed-ubuntu22.04-cuda12-x86_64.tbz
+  CLANG_VER: 17
+  MLNX_OFED_VER: 23.10-1.1.9.0
```

Comment:
Consider updating to MOFED 24.10
https://network.nvidia.com/products/infiniband-drivers/linux/mlnx_ofed/

Reply:
done